Adafruit_USBD_CDC.h

```diff
@@ -40,6 +40,8 @@ class Adafruit_USBD_CDC : public Stream, Adafruit_USBD_Interface
 	void begin(uint32_t baud_count);
 	void begin(uint32_t baud, uint8_t config);
 	void end(void);
+        uint32_t getBaud() { return _baud_rate; }
+        void setBaud(uint32_t baud) {  _baud_rate = baud; }
```

Comment:
These aren't Arduino std AP, it got ignored anyway. I don't see why we would need it